config/src/tables.lua

```diff
@@ -0,0 +1,71 @@
+-- Copyright 2018 Yahoo Inc.
+-- Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+local parser = require("utils.jsonParser")
+local table_utils = require("utils.tableUtils")
+
+local M = {
+    physicalTables = {},
+    logicalTables = {}
+}
+
+-------------------------------------------------------------------------------
+-- Default
+-------------------------------------------------------------------------------
+
+DEFAULT = {
+    ALL_DIM = {
```

Comment:
Shouldn't we be deriving these from the `metrics` and `dimensions` modules?

Keeping these up to date with those two modules would be a pain.